mrjones2014/yaclt
# Add option + middleware handler to set message from last commit message

Resolves: #82 

Adds an option, which is pre-processed by a middleware handler, that allows the message to be pulled from the first line of the most recent commit message.

# How to Test

1. Run `yarn build`
2. Run `git log -n 1` to check what the most recent commit message is
3. Run `./dist/index.js new --changeType NEW --lastCommit`
4. It should not error and it should generate a changelog entry with the last commit message as the changelog message
